build.gradle.kts

```diff
@@ -25,14 +25,27 @@ java {
 }
 
 tasks {
-  withType<Test>().configureEach {
-    useJUnitPlatform()
-  }
   withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
     kotlinOptions {
       jvmTarget = "18"
     }
   }
+
+  register<Test>("unitTest") {
+    useJUnitPlatform()
+
+    filter {
+      excludeTestsMatching("uk.gov.justice.digital.hmpps.hmppsintegrationapi.smoke*")
+    }
+  }
+
+  register<Test>("smokeTest") {
+    useJUnitPlatform()
```

Comment:
Is `useJUnitPlatform()` still required?

Reply:
Good spot, forgot to check - nope it isn't!